### PR TITLE
Fixed position flash alerts

### LINKED ISSUE #256: Fixed position flash div
Add a fixed position div for flash messages to the bulk edit view. Currently, user has to scroll up to the top of the page when there are lots of files in a scanned resource.

Should this be expanded to all pages?

### PR DESCRIPTION
**Pull request by eliotjordan**
Positions flash alert div in a fixed position at the top of the page. Also moves javascript alerts to the same position. Closes #256.

### CODE REVIEW

File: app/assets/javascripts/sortable.es6 [hunk 1]
```diff
@@ -40,7 +40,6 @@
         } else {
           this.flash.set("danger", "Unable to save new order.")
         }
-        this.disable()
```
Comment (eliotjordan) [inline app/assets/javascripts/sortable.es6]: Not sure why this doesn't work, but it raises an error in the browser. Removing it doesn't seem to change functionality.

**Comment by jpstroop** [comment]
I think we should wait on this now until after the demo :ok_hand: ?

**Comment by tpendragon** [comment]
:+1: to waiting. `sortable.es6` is dangerous territory.

**Comment by tpendragon** [comment]
@jpstroop @escowles This changes -all- flash alerts to be fixed at the top. Is that desirable behavior?

@eliotjordan At the very least it seems like the height's a little big.

**Comment by escowles** [comment]
:+1: to making all flash alerts fixed at the top -- doing this consistently across all views seems better than having some be fixed and some inline.

**Comment by eliotjordan** [comment]
@tpendragon Height suggestion?

**Comment by tpendragon** [comment]
@eliotjordan Maybe a padding-top/padding-bottom on the .alert of only 5px, instead of 25?

**Comment by eliotjordan** [comment]
@tpendragon I'll give it a go.

**Comment by eliotjordan** [comment]
Padding reduced to 5px.

**Pull request merged**